### DISCUSSION TITLE
What table games have you played?

### DISCUSSION THREAD

**Discussion by OguniShun** [NAIST-SE/AttractingContributors#542] [General]
Many of us have played Table games.
There are playing cards, Othello, and many others.
Table games are fun for one person or for multiple people.
Can you tell me about a table game you have played?

多くの方がテーブルゲームをプレイしたことがあると思います。
トランプ、オセロなど多くのゲームがあります。
いろんな人とプレイするテーブルゲームは面白いですよね。
あなたがプレイしたことのあるテーブルゲームを教えてくれませんか？

https://github.com/OguniShun/AttractingContributors_Table-Game

**Comment by ani-hovhannisyan** [comment]
I added #2, please help me with my project too:
https://github.com/ani-hovhannisyan/AttractingContributors-Motorcycle-Bicycle.git
